modules/core/localizer.js

```diff
@@ -148,19 +148,21 @@ export function coreLocalizer() {
         /** @type {string[]} */
         let toUse = [];
         for (const locale of requestedLocales) {
-            if (supportedLocales[locale]) toUse.push(locale);
+            const safeLocale = locale.split(';')[0].trim(); // strip any quality values like ";q=0.5"
```

Comment:
Is there some way to fold this into where we process the preference, so other code doesn’t behave inconsistently?

https://github.com/openstreetmap/iD/blob/13b96eb391068c9532b5eca18f67244a1315a8cc/modules/core/localizer.js#L72-L73